### PR TITLE
Pass on 403 errors received from content-store.

### PR DESCRIPTION
**Pull request by danielroseman**
Catches the HttpForbidden error that is now raised by gds-api-adapters,
and renders a 403 response.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+
+/public/government-frontend
```
Comment (alext) [inline .gitignore]: Did you mean to add this?  It seems unrelated to this PR.

File: app/controllers/content_items_controller.rb [hunk 2]
```diff
@@ -9,6 +9,8 @@ def show
     else
       render text: 'Not found', status: :not_found
     end
+  rescue GdsApi::HTTPForbidden => e
+    render text: e.message, status: 403
```
Comment (alext) [inline app/controllers/content_items_controller.rb]: This might be better handled with a `rescue_from` call at controller level.  See https://github.com/alphagov/collections/blob/master/app/controllers/subtopics_controller.rb#L6 for an example (in that example, `error_404` is a method defined in `ApplicationController`).

File: .gitignore (hunk 1 above)
Comment (danielroseman) [inline .gitignore]: Sorry, removed.

File: app/controllers/content_items_controller.rb (hunk 2 above)
Comment (danielroseman) [inline app/controllers/content_items_controller.rb]: Done.
